claim-check/code-samples/sample-3/readme.md

```diff
@@ -1,108 +1,107 @@
 
-* a resource group
-* a V2 storage account
-* a storage account container
-* a service bus namespace, service bus and service bus queue
-* a function app in an app service plan
-* an application insights service
+   ```shell
+   sed "s/{EVENT_HUBS_NAMESPACE}/evhns-${NAME_PREFIX}/g" ClientConsumer2/appsettings.json.template >ClientConsumer2/appsettings.json
+   sed -i "s/{EVENT_PROCESSOR_STORAGE_ACCOUNT_NAME}/st${NAME_PREFIX}ehub/g" ClientConsumer2/appsettings.json
 
-## Running the sample
+   sed "s/{EVENT_HUBS_NAMESPACE}/evhns-${NAME_PREFIX}/g" FunctionConsumer2/local.settings.json.template > FunctionConsumer2/local.settings.json
+   ```
 
-The Azure Function is used to demonstrate a client application that acts as the initial processor for the large payload. This could be replaced with any other application with similar code
+1. [Run Azurite](https://learn.microsoft.com/azure/storage/common/storage-use-azurite#run-azurite) blob storage emulation service.
 
-Use the Azure Function URL obtained from the portal as shown below
+   > The local storage emulator is required as an Azure Storage account is a required "backing resource" for Azure Functions.
 
-![Get Azure Function URL](images/Function_GetURL.jpg)
+1. Launch the consumer sample application that will process the claim check messages from Service Bus.
 
-Send a post request on the URL obtained using a message which acts as your large payload
+   Two applications are provided as sample that illustrate consuming the claim check message sent via Azure Event Hubs: one implemented as a Command Line Interface (CLI) application, and the other one implemented as an Azure Function, showcasing the serveless approach.
 
-```bash
-curl -H "Content-Type: application/json" -X POST https://<yourapp>.azurewebsites.net/api/ClaimCheck -d "This is the large payload test"
-```
+   Run the desired sample application, CLI or Function to connect to the messaging system and process messages as they arrive.
 
-or, if you want to try sending something bigger:
+   1. For the CLI sample
 
-```bash
-curl -H "Content-Type: application/json" -X POST https://<yourapp>.azurewebsites.net/api/ClaimCheck -d @readme.md
-```
+   ```bash
+   dotnet run --project sample-2\ClientConsumer3
+   ```
 
-The Azure Function code uses the Service Bus plugin which drops the payload as a blob in the Storage Account created earlier and also sends a notification to consumer client using Service Bus Queue.
+   1. For the Function sample
 
-The script `sample-3-azure-setup.sh` had automatically configured `App.config` so that the consumer application points to the created resources. Run the consumer application locally:
+   ```bash
+   cd sample-2\FunctionConsumer3
+   func start
+   ```
 
-```bash
-cd client-consumer
-dotnet run
-```
+  > Please note: For demo purposes, the console application prints payload content on the screen. So keep that in mind if you want to try sending really large payloads.
```

Comment:
Shouldn't this note just be under the console one, not the function one?  Also, unless I missed it, it's just logging to Information.  Are those printed to the screen?  (I didn't run this one to check)